CIP-0095/README.md

```diff
@@ -123,24 +123,23 @@ extending the API (as a plain integer, without padding). For example:
 
 #### CIP-95 Data Types
 
-##### PubDRepKey
+##### DRepID
 
 ```ts
-type PubDRepKey = string;
+type DRepID = string;
 ```
 
-A hex-encoded string representing 32 byte Ed25519 DRep public key, as described
-in [CIP-0105 | Conway Era Key Chains for HD Wallets](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0105/README.md).
+A hex-encoded string representing a Blake2b-224 hash digest of a 32 byte
+Ed25519 public DRep key.
```

Comment:
shouldn't we make a comment here that this can also be in CIP129 format?

Reply:
This is nothing but breaking standards and introducing a lifelong wart in the ecosystem.

Reply:
I do intend to get back to this one and sort it out, amoung other CIP95 housekeeping

thanks for the nudge and the good points @ashisherc @rphair 🤝

Reply:
OK... relaxing status from `Last Check` (a packed section in tomorrow's meeting) & let us know when it's finally ready 😎